.ci/test_r_package_windows.ps1

```diff
@@ -39,12 +39,24 @@ function Remove-From-Path {
 # remove some details that exist in the GitHub Actions images which might
 # cause conflicts with R and other components installed by this script
 $env:RTOOLS40_HOME = ""
+Remove-From-Path ".*Amazon.*"
+Remove-From-Path ".*Anaconda.*"
+Remove-From-Path ".*android.*"
+Remove-From-Path ".*Android.*"
 Remove-From-Path ".*chocolatey.*"
 Remove-From-Path ".*Chocolatey.*"
-Remove-From-Path ".*Git.*mingw64.*"
+Remove-From-Path ".*\\Git\\.*"
```

Comment:
After cloning a repo we don't need Git in R-package jobs.
There are a lot of possible conflicts in git folder.

![image](https://user-images.githubusercontent.com/25141164/125173092-edd68d00-e1c5-11eb-9072-9a85dc1195a2.png)